sources/bloodhound/src/bin/bottlerocket-checks/checks.rs

```diff
@@ -0,0 +1,1280 @@
+            "kernel.unprivileged_bpf_disabled = 1",
+        ) {
+            if !found {
+                result.error = "Unprivileged eBPF is not disabled".to_string();
+                result.status = results::CheckStatus::FAIL;
+            } else {
+                result.status = results::CheckStatus::PASS;
+            }
+        } else {
+            result.error = "unable to verify kernel.unprivileged_bpf_disabled setting".to_string();
+        }
+
+        result
+    }
+
+    fn metadata(&self) -> results::CheckerMetadata {
+        results::CheckerMetadata {
+            title: "Ensure unprivileged eBPF is disabled".to_string(),
+            id: "1.4.3".to_string(),
+            level: 1,
+            name: "br01040300".to_string(),
+            mode: results::Mode::Automatic,
+        }
+    }
+}
+
+// =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<=
+
+pub struct BR01040400Checker {}
+
+impl results::Checker for BR01040400Checker {
+    fn execute(&self) -> results::CheckerResult {
+        let mut result = results::CheckerResult {
+            error: String::new(),
+            status: results::CheckStatus::SKIP,
+        };
+
+        if let Some(found) = look_for_string_in_output(
+            SYSCTL_CMD,
+            ["user.max_user_namespaces"],
+            "user.max_user_namespaces = 0",
+        ) {
+            if !found {
+                result.error = "User namespaces are not disabled".to_string();
+                result.status = results::CheckStatus::FAIL;
+            } else {
+                result.status = results::CheckStatus::PASS;
+            }
+        } else {
+            result.error = "unable to verify user.max_user_namespaces setting".to_string();
+        }
+
+        result
+    }
+
+    fn metadata(&self) -> results::CheckerMetadata {
+        results::CheckerMetadata {
+            title: "Ensure user namespaces are disabled".to_string(),
+            id: "1.4.4".to_string(),
+            level: 2,
+            name: "br01040400".to_string(),
+            mode: results::Mode::Automatic,
+        }
+    }
+}
+
+// =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<= =>o.o<=
+
+pub struct BR01050100Checker {}
+
+impl results::Checker for BR01050100Checker {
+    fn execute(&self) -> results::CheckerResult {
+        let mut result = results::CheckerResult {
+            error: String::new(),
+            status: results::CheckStatus::SKIP,
+        };
+
+        // Trying to avoid bringing in regex for now
+        let to_match = &vec![
+            ("SELinux status: ", " enabled"),
+            ("Loaded policy name: ", " fortified"),
+            ("Current mode: ", " enforcing"),
+            ("Mode from config file: ", " enforcing"),
+            ("Policy MLS status: ", " enabled"),
+            ("Policy deny_unknown status: ", " denied"),
+            ("Memory protection checking: ", " actual (secure)"),
+        ];
```

Comment:
You could potentially have two versions of your `check_for` helper functions, one that takes a single line that must be in the output, and one that takes multiple lines, and have the single line one call into the multi-line function.

This would also be useful in the dm-verity checker, and in the `sysctl` checker, since you can query multiple sysctls at once:
```
❯ sysctl net.ipv4.conf.all.send_redirects net.ipv4.conf.default.send_redirects
net.ipv4.conf.all.send_redirects = 0
net.ipv4.conf.default.send_redirects = 1
```